dockstore-client/src/main/java/io/dockstore/client/cli/nested/AbstractEntryClient.java

```diff
@@ -1177,45 +1138,87 @@ private void wesServiceInfo(WorkflowExecutionServiceApi clientWorkflowExecutionS
         }
     }
 
+    /**
+     * Given the parsed command object, determine if we are to print help commands
+     * @param wesCommandParser Parse commands
+     * @return
```

Comment:
If you keep the signature this way, fill out the `@return`

Reply:
Updated: https://github.com/dockstore/dockstore-cli/pull/99/commits/3bb5f2c817276fb79599208aea88474729fba840